tests/Validation/ValidationValidatorTest.php

```diff
@@ -1893,7 +1893,7 @@ public function testValidateImplicitEachWithAsterisksForRequiredNonExistingKey()
         $this->assertFalse($v->passes());
 
         $data = ['people' => [
-            ['cars' => [['model' => 2005], []]],
+            ['cars' => [['model' => 2005], ['name' => 'alpha']]],
```

Comment:
Please don't modify this test. We need to be able to assert that it still works without modification.